src/App.tsx

```diff
@@ -12,31 +13,9 @@ export const tabs = [
 export const App: React.FC = () => {
```

Comment:
Add this requirement to code
![image](https://github.com/mate-academy/react_tabs/assets/70009205/89e2b4fc-450a-4962-9afb-7a25e71efc38)